website/package.json

```diff
@@ -32,25 +32,25 @@
     ]
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0",
-    "@docusaurus/plugin-client-redirects": "^2.0.0",
-    "@docusaurus/plugin-pwa": "^2.0.0",
-    "@docusaurus/preset-classic": "^2.0.0",
-    "@docusaurus/remark-plugin-npm2yarn": "^2.0.0",
+    "@docusaurus/core": "0.0.0-5658",
+    "@docusaurus/plugin-client-redirects": "0.0.0-5658",
+    "@docusaurus/plugin-pwa": "0.0.0-5658",
+    "@docusaurus/preset-classic": "0.0.0-5658",
+    "@docusaurus/remark-plugin-npm2yarn": "0.0.0-5658",
     "clsx": "^2.0.0",
-    "docusaurus-remark-plugin-tab-blocks": "^1.2.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "docusaurus-remark-plugin-tab-blocks": "^2.0.0-beta",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "react-github-btn": "^1.3.0",
     "react-lite-youtube-embed": "^2.2.2",
     "react-markdown": "^8.0.0"
   },
   "devDependencies": {
     "@babel/core": "^7.11.6",
     "@crowdin/cli": "^3.5.2",
-    "@docusaurus/module-type-aliases": "^2.0.0",
+    "@docusaurus/module-type-aliases": "0.0.0-5658",
     "@tsconfig/docusaurus": "^1.0.5",
```

Comment:
maybe bump this as well?

Reply:
What do you mean? bump to what value?

Reply:
v2 has been released

Reply:
??? but it's a Docusaurus dependency, we want it to be v3 (canary) like the other docusaurus dependencies. 

All official Docusaurus deps should use the exact same versions.

Reply:
https://github.com/tsconfig/bases/blob/main/bases/docusaurus.json

Reply:
Ahh! thought you were talking about the module above.

Actually you are right I forgot something, we are using a new official package `@docusaurus/tsconfig` now

Reply:
just remove it based on https://github.com/tsconfig/bases/issues/196?

Reply:
> Ahh! thought you were talking about the module above.
> 
> Actually you are right I forgot something, we are using a new official package `@docusaurus/tsconfig` now

👍